packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -410,6 +426,7 @@ onMounted(async () => {
 		// Set default output which is the input (original model)
 		createOutput(originalModel);
 	}
+	hasCodeChange();
```

Comment:
why is this checked for in the onMounted()?

Reply:
Not needed removed